hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/schema/TableResolverImpl.java

```diff
@@ -148,19 +150,24 @@ private Mapping resolveMapping(Mapping mapping) {
         } else {
             connector = connectorCache.forType(type);
         }
+        String objectType = mapping.objectType() == null
```

Comment:
what will happen for mappings created in 5.2 release when performing rolling upgrade? will we use `defaultObjectType`?

Reply:
should be fine and use default